test/Microsoft.TemplateEngine.EndToEndTestHarness/Program.cs

```diff
@@ -198,8 +205,9 @@ private static void FirstRun(IEngineEnvironmentSettings environmentSettings, IIn
 
             string packages = Path.Combine(dir, "..", "..", "..", "..", "..", "artifacts", "packages") + Path.DirectorySeparatorChar + "*";
             string templates = Path.Combine(dir, "..", "..", "..", "..", "..", "template_feed") + Path.DirectorySeparatorChar;
+            string testTemplates = Path.Combine(dir, "test_templates") + Path.DirectorySeparatorChar;
             installer.InstallPackages(new[] { packages });
-            installer.InstallPackages(new[] { templates });
+            installer.InstallPackages(new[] { templates, testTemplates });
```

Comment:
Should we maybe not install the real templates for tests? Changes to them could conceivably cause side-effects in the tests.